tests/integration/test_s3_cluster/test.py

```diff
@@ -72,7 +72,7 @@ def started_cluster():
             "s0_0_0",
             main_configs=["configs/cluster.xml", "configs/named_collections.xml"],
             user_configs=["configs/users.xml"],
-            macros={"replica": "node1", "shard": "shard1"},
+            macros={"replica": "replica1", "shard": "shard1"},
```

Comment:
Why change this?

Reply:
Just to unification with other two nodes below, they have `"replica": ""replicaX"` macro.